### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Moses-chibuike**
- Add any missing JSX elements to your project (without adding extra functionality)
- Final design should match Zeplin's design (this includes all elements + the same styling)
- Use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/BookItem.js [hunk 1]
```diff
@@ -3,27 +3,55 @@ import { useDispatch } from 'react-redux';
...
+          </div>
+        </div>
+        <div className="progress-section-divider" />
+        <div className="current-chapter-section">
+          <span className="cur-chapter-title">CURRENT CHAPTER</span>
+          <h4 className="cur-chapter">Chapter 10</h4>
+          <button type="button" className="update-progress-btn">UPDATE PROGRESS</button>
+        </div>
+      </div>
     </div>
```
Comment (DammyShittu) [inline src/components/BookItem.js]: - [ ] Kindly note that we are to style the app to match the design EXACTLY. Kindly improve your bookitem styling to match the design. Kindly use the correct font styling.

- [ ] Kindly move the `Update Progress` section on the right demarcation more towards the left to match to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  kisi

File: src/components/Navbar.js [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
+              </li>
+            </ul>
+          </nav>
 
-        <Outlet />
+          <Outlet />
+        </div>
       </div>
+      <FaUserAlt className="icon" />
     </div>
```
Comment (DammyShittu) [inline src/components/Navbar.js]: - [ ] Kindly improve the font of the navbar section to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  siisos

File: src/components/Bookform.js [hunk 3]
```diff
@@ -39,11 +42,14 @@ function Bookform() {
...
-        <input type="text" placeholder="Add Author's name..." name="author" value={formData.author} onChange={setValue} />
-        <button type="submit">Add Book</button>
+        <div className="form-actions">
+          <input type="text" placeholder="Add book name..." name="title" value={formData.title} onChange={setValue} />
+          <input type="text" placeholder="Add Author's name..." name="author" value={formData.author} onChange={setValue} />
+          <button type="submit">Add Book</button>
+        </div>
       </form>
     </div>
   );
```
Comment (DammyShittu) [inline src/components/Bookform.js]: - [ ] Kindly improve the font of the form component to match the design.

  Link to the design: https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md

  jsisiao

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Moses-chibuike,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @Moses-chibuike  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Used gitflow :+1: 
Great effort in making the requestec changes :+1: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Bookform.js (hunk 3 above)
Comment (Moses-chibuike) [reply src/components/Bookform.js]: Ok well noted

File: src/components/Navbar.js (hunk 2 above)
Comment (Moses-chibuike) [reply src/components/Navbar.js]: Ok well noted

File: src/components/BookItem.js (hunk 1 above)
Comment (Moses-chibuike) [reply src/components/BookItem.js]: Ok well noted

**Pull request merged**